src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -705,7 +711,10 @@ private function createHashers(array $hashers, ContainerBuilder $container)
         ;
     }
 
-    private function createHasher(array $config)
+    /**
+     * @return Reference|array{class: class-string, arguments: list<mixed>}
```

Comment:
not only class+arguments - not sure it's worth listing them all?

Reply:
Right, I missed the spot where the array is returned as-is. I'll change it to `array<string, mixed>`.